Add docs on applying the govuk brand refresh (#291)
alphagov/govuk-prototype-kit-docs: docs/v13/documentation/use-govuk-brand-refresh.md

---
heading: Use the GOV.UK brand refresh in your prototype
title: Use the GOV.UK brand refresh in your prototype
---

> On 25th June 2025, [GDS launched the refreshed GOV.UK brand for web](https://www.gov.uk/guidance/changes-to-govuk). If you are prototyping a GOV.UK service then it's recommended that you use the refreshed GOV.UK brand for consistency with GOV.UK and other GOV.UK services.
> 
> You can do this in your prototype via the GOV.UK Frontend prototype kit plugin. Make sure that your prototype is using at least GOV.UK Frontend version 5.10.0. See the guidance on [installing and using plugins](/docs/install-and-use-plugins) for details on how to check this.
> 
> Go to your prototype's `config.json`, located in your prototype's `app` folder. Add a comma after the `serviceName` line if there isn't one there already. After `serviceName` but still within the curly brackets, enter the following:

Reviewer: A few style fixes and a proposed approach – could we talk about this the same way we talk about the feature flag elsewhere?

```suggestion
On 25 June 2025, [GDS launched a refreshed GOV.UK brand for web](https://www.gov.uk/guidance/changes-to-govuk). If you're prototyping a GOV.UK service, we recommend you use the refreshed brand so it's consistent with the GOV.UK website and other GOV.UK services.

You can update the branding in your prototype using the `rebrand` option in the GOV.UK Frontend Prototype Kit plugin.

Before you start, make sure your prototype is using GOV.UK Frontend v5.10.0 or later.  See [installing and using plugins](/docs/install-and-use-plugins) for details on how to check.

## Add the `rebrand` option and set it to `true`

Go to your prototype's `config.json`, located in your prototype's `app` folder. Add a comma after the `serviceName` line if there isn't one there already. After `serviceName` but still within the curly brackets, enter the following:
```

```json
"plugins": {
  "govuk-frontend": {
    "rebrand": true
  }
}
```

If there was nothing else besides `serviceName` in your config, it should now look like this:

```json
{
  "serviceName": "Your service name",
  "plugins": {
    "govuk-frontend": {
      "rebrand": true
    }
  }
}
```

> Restart your prototype. You should now see the refreshed GOV.UK brand.

Reviewer 2: just checked and it restarts automatically, though you do need to refresh I think

Author: What if I amended this to:

```suggestion
Refresh your browser or restart your prototype. You should now see the refreshed GOV.UK brand.
```

Author: Maybe I can just scrap restart your prototype? It's sort of creating an assumption that your prototype might not auto-refresh which we don't expect to be the case.

Author: I've gone with a link to the starting the kit docs for simplicity

You can check if your prototype is using the refreshed GOV.UK brand using [the guidance on changes to GOV.UK](https://www.gov.uk/guidance/changes-to-govuk). If your prototype is using the refreshed brand then you don't need to make any changes.

You can update the branding in your prototype using the `rebrand` option in the GOV.UK Frontend Prototype Kit plugin.

Before you start, make sure your prototype is using GOV.UK Frontend v5.10.0 or later.  See [installing and using plugins](/docs/install-and-use-plugins) for details on how to check.

## Add the `rebrand` option and set it to `true`

Go to your prototype's `config.json`, located in your prototype's `app` folder. Add a comma after the `serviceName` line if there isn't one there already. After `serviceName` but still within the curly brackets, enter the following:

```json
"plugins": {
  "govuk-frontend": {
    "rebrand": true
  }
}
```

If there was nothing else besides `serviceName` in your config, it should now look like this:

```json
{
  "serviceName": "Your service name",
  "plugins": {
    "govuk-frontend": {
      "rebrand": true
    }
  }
}
```

> Start your prototype locally if you haven't already, [as per the instructions on running the kit](/docs/install/how-to-run-the-kit). You should now see the refreshed GOV.UK brand.

Reviewer: ````suggestion
---
heading: Use the GOV.UK brand refresh in your prototype
title: Use the GOV.UK brand refresh in your prototype
---

On 25 June 2025, GDS launched a refreshed GOV.UK brand for web. If you're prototyping a GOV.UK service and your prototype is not using the refreshed GOV.UK brand, we recommend you use the refreshed brand so that it's consistent with the GOV.UK website and other GOV.UK services.

To check if your prototype is using the refreshed GOV.UK brand, see [the guidance on changes to GOV.UK](https://www.gov.uk/guidance/changes-to-govuk). If your prototype is already using the refreshed brand, you do not need to make any changes.

You can update the branding in your prototype using the `rebrand` option in the GOV.UK Frontend Prototype Kit plugin.

Before you start, make sure your prototype is using GOV.UK Frontend v5.10.0 or later.  See [installing and using plugins](/docs/install-and-use-plugins) for details on how to check.

## Add the `rebrand` option and set it to `true`

Go to your prototype's `config.json`, located in your prototype's `app` folder. Add a comma after the `serviceName` line if there isn't one there already. After `serviceName` but still within the curly brackets, enter the following:

```json
"plugins": {
  "govuk-frontend": {
    "rebrand": true
  }
}
```

If there was nothing else besides `serviceName` in your config, it should now look like this:

```json
{
  "serviceName": "Your service name",
  "plugins": {
    "govuk-frontend": {
      "rebrand": true
    }
  }
}
```

If you haven't already, start your prototype locally, [using the instructions on running the kit](/docs/install/how-to-run-the-kit). You should now see the refreshed GOV.UK brand.
````
